main.tf

```diff
@@ -91,7 +91,7 @@ resource "aws_redshift_parameter_group" "default" {
 
 module "logging_bucket" {
   count          = var.logging ? 1 : 0
-  source         = "github.com/schubergphilis/terraform-aws-mcaf-s3?ref=v0.5.0"
+  source         = "github.com/schubergphilis/terraform-aws-mcaf-s3?ref=v0.7.0"
```

Comment:
Hey @Vino-Bala why not upgrade to the latest version? https://github.com/schubergphilis/terraform-aws-mcaf-s3/releases . You can also use the public registry for thsi module by the way: https://registry.terraform.io/modules/schubergphilis/mcaf-s3/aws/latest

Reply:
Indeed I can use the latest version (v0.10.0). 
I had issue while creating new redshift cluster, "Error: error creating S3 bucket ACL for gmt-datalayer-dev-redshift-log-shared-vpc: AccessControlListNotSupported: The bucket does not allow ACLs ..."
So to fix this part I used v0.7.0.